components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/API.java

```diff
@@ -587,4 +588,20 @@ public boolean isDefaultVersion(){
     public boolean isPublishedDefaultVersion(){
         return isPublishedDefaultVersion;
     }
+
+    public String getAllowedOrigins() {
```

Comment:
We save the origins in a comma separated list therefore we don't need a String array in representation